docs/manuals/gui/viselements/index.md

```diff
@@ -109,6 +109,60 @@ fragment similar to:
     If you set a property that a visual element does not recognize, it is
     ignored without any warning.
 
+!!! important "Indentation and block elements: element tag identifiers"
+    Markdown depends heavily on text indentation to decide whether or not a new paragraph or section
+    should be created.<br/>
+    When dealing with block elements to create sections on your page, you might be
+    tempted to indent the opening element tags, so the Markdown text is easier to read.<
+
+    The following Markdown content:
+    ```
+    <|
+    The part content
+    <|
+    First sub-part content.
+    |>
+    <|
+    Second sub-part content.
+    |>
+    |>
+    ```
+    would be easier to read if indented, as shown here:
+    ```
+    <|
```

Comment:
It works with 2 spaces...

Reply:
Ok. I'll complete this sentence saying that it might work but...
Works for you?